prow/jobs/kyma/tests/integration/event-service/test-event-service.yaml

```diff
@@ -76,11 +76,11 @@ presubmits: # runs on PRs
 
 postsubmits:
   kyma-project/kyma:
-    - name: post-master-kyma-tests-gateway-tests
+    - name: post-master-kyma-tests-integration-event-service-tests
```

Comment:
So again, IMO `post-master-kyma-tests-integration-event-service` is sufficient

Reply:
Done.